packages/emails/src/templates/BrokenIntegrationEmail.tsx

```diff
@@ -0,0 +1,117 @@
+import { TFunction } from "next-i18next";
+import { Trans } from "react-i18next";
+
+import { AppStoreLocationType } from "@calcom/app-store/locations";
+import { WEBAPP_URL } from "@calcom/lib/constants";
+import type { CalendarEvent, Person } from "@calcom/types/Calendar";
+
+import { BaseScheduledEmail } from "./BaseScheduledEmail";
+
+// https://stackoverflow.com/questions/56263980/get-key-of-an-enum-from-its-value-in-typescript
+export function getEnumKeyByEnumValue(myEnum: any, enumValue: number | string): string {
+  const keys = Object.keys(myEnum).filter((x) => myEnum[x] == enumValue);
+  return keys.length > 0 ? keys[0] : "";
+}
+
+const BrokenVideoIntegration = (props: { location: string; eventTypeId?: number | null; t: TFunction }) => {
+  return (
+    <Trans i18nKey="broken_video_action" t={props.t} values={{ location: props.location }}>
+      We could not add the <span>{props.location}</span> meeting link to your scheduled event. Contact your
+      invitees or update your calendar event to add the details. You can either&nbsp;
+      <a
+        href={
+          props.eventTypeId ? `${WEBAPP_URL}/event-types/${props.eventTypeId}` : `${WEBAPP_URL}/event-types`
+        }>
+        change your location on the event type
+      </a>
+      &nbsp;or&nbsp;
+      <a href={`${WEBAPP_URL}/apps/installed`}>removing and adding the app again.</a>
+    </Trans>
+  );
+};
+
+const BrokenCalendarIntegration = (props: {
+  calendar: string;
+  eventTypeId?: number | null;
+  t: TFunction;
+}) => {
+  const { t } = props;
+
+  return (
+    <Trans i18nKey="broken_calendar_action" t={t}>
+      We could not update your <span>{props.calendar}</span>.{" "}
+      <a href={`${WEBAPP_URL}/apps/installed`}>
+        Please check your calendar settings or remove and add your calendar again
+      </a>
+    </Trans>
+  );
```

Comment:
Made some fixes on the `BrokenVideoIntegration` let's see if you can replicate in here [based on the docs](https://react.i18next.com/latest/trans-component)